.travis.yml

```diff
@@ -101,6 +101,36 @@ matrix:
         - CC1=gcc-6
         - CXX1=g++-6
 
+    - os: linux
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - g++-6
+      env:
+        - FAVORITE_CONFIG=no
+        - LLVM_VERSION="4.0.0"
+        - LLVM_CONFIG="llvm-config-4.0"
+        - config=debug
+        - CC1=gcc-6
+        - CXX1=g++-6
+
+    - os: linux
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - g++-6
+      env:
+        - FAVORITE_CONFIG=yes
```

Comment:
can only have 1 favorite config. can you set this to "no" for now.

favorite config is used to decide which linux version will be done for release. so until we decide that 4.0.0 is the default, it should be "no"